js/tinymce/plugins/accordion/plugin.js

```
@@ -0,0 +1,1057 @@
+      if (x === null) {
+        return 'null';
+      } else if (t === 'object' && Array.isArray(x)) {
+        return 'array';
+      } else if (t === 'object' && hasProto(x, String, (o, proto) => proto.isPrototypeOf(o))) {
+        return 'string';
+      } else {
+        return t;
+      }
+    };
+    const isType$1 = type => value => typeOf(value) === type;
+    const isSimpleType = type => value => typeof value === type;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)